stack/examples/flake8-travis.yml

```diff
@@ -6,3 +6,19 @@ matrix:
       install:
       - pip install flake8
       script: flake8
+        include:
+    - env: TEST=shellcheck
+      services:
+        - docker
+      script: |
+        set -e
+        shopt -s globstar nullglob
+        CHECK=( **/*.sh )
+        [[ ${#CHECK[@]} -eq 0 ]] && exit
+        docker run -v $(pwd):$(pwd) -w $(pwd) \
+          koalaman/shellcheck-alpine:v0.4.6 -x "${CHECK[@]}"
+branches:
+  only:
+  - master
+notifications:
```

Comment:
I don't like seeing this decision (turning off email notifications) backdoored into a single commit with an unrelated change.

I'm happy with this change personally, but this needs to be in a PR that's not titled "add shellcheck to .travis.yml example."

Reply:
I've split the branch restriction and disabling email into #215.